### DISCUSSION TITLE
透過自然語言叫 AI 辦事

### DISCUSSION THREAD

**Discussion by Hchejxnc** [1101-GS4719-C/Python-Community-GS4719-C#577] [General]
> Google 前員工 David Luan 於週二成立了 AI 人工智慧研發實驗室 Adept AI，該公司成立的宗旨極具野心：教機器如何使用「全世界所有軟體與 API」。身兼執行長的 David Luan 更清楚的表明，Adept AI 是一間專門打造通用人工智慧的研發與產品實驗室。

> Luan 夢想未來人們可以直接透過自然語言來指示 AI 模型辦事，這樣的系統除了學習如何透過可用的軟體來完成任務外，最關鍵的是要教會系統如何使用軟體與 API，如此一來系統才會主動執行人類的指令。

> 如果未來能透過講話直接去控制AI，那感覺會很方便。

文章來源：https://technews.tw/2022/04/30/ex-googlers-take-a-stab-at-building-general-intelligence-that-makes-software-do-what-you-tell-it/

**Comment by hapiwangy** [comment]
這感覺像是一個漫長的過程，因為這牽涉到許多領域，光是自然語言處裡就是一大難題了

**Comment by yenwei12138** [comment]
若真的成真的話，可能在未來工程師們的工作都要被取代了QQ

**Comment by ghost** [comment]
是也不到取代啦XD，而且這段路確實有段很長的路要走

**Comment by bloodtrail0511** [comment]
直接用講話去控制AI，那我現在學的程式不就沒啥作用了QQ

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，期望AI能與人類一起成長。

**Comment by PoooYen** [comment]
又是一段漫漫長路，但我非常期待

**Comment by d26488399** [comment]
希望不要有人叫AI寫病毒

**Comment by Yuhsin000** [comment]
既期待又害怕受傷害，想試用又怕被取代QQ

**Comment by cordydeng** [comment]
如果這項技術能夠成功，想必可以降低AI取代人們工作的趨勢，讓我們能夠不被AI控制

**Comment by sys511613** [comment]
自然語言處理已經做得越來越厲害，相信電腦能完全聽懂人類語言的日子不遠了，不像現在的siri只會裝傻

**Comment by Jay-722** [comment]
謝謝分享，科技越來越發達了

**Comment by SamTU0222** [comment]
謝謝分享 技術真的是會進步的神速

**Comment by handsomeniko** [comment]
自然與語言處理真的很有趣

**Comment by lx3211** [comment]
以後的AI會更人性化吧
siri真的只會裝傻:)

**Reply by Rosancl** [reply]
好好笑 siri只會裝傻xddd

**Comment by Rosancl** [comment]
期待那天的到來，那時候的環境大概就已經是未來科技影片中會出現的樣子了

**Comment by Sazmafuji** [comment]
AIDUNGEON，原來這就是自然語言處理

**Comment by yucheng1028** [comment]
這樣ai會取代更多人力

**Comment by henrywen547** [comment]
好期待這項技術能成功!
但也怕會有錯誤使用的問題
